Terminal.Gui/Views/Menu/MenuBar.cs

```diff
@@ -1591,4 +1591,177 @@ private MenuBar GetMouseGrabViewInstance (View view)
     }
 
     #endregion Mouse Handling
+
+
+    /// <inheritdoc />
+    public bool EnableForDesign<TContext> (in TContext context) where TContext : notnull
+    {
+        if (context is not Func<string, bool> actionFn)
+        {
+            actionFn = (s) => true;
+        }
+
+        Menus =
+        [
+            new MenuBarItem (
+                             "_File",
+                             new MenuItem []
```

Comment:
Can use a collection initializer expression here, instead of the array constructor, for free compiler optimizations, if you want, just like is already being done for the `Menus` property already. Might as well. :man_shrugging: 

Plus that'll result in less nesting, which I guess is a bonus.

Reply:
Menu is getting re-written. Not going to touch this code until then.